src/mlpack/methods/sparse_autoencoder/sparse_autoencoder_impl.hpp

```diff
@@ -72,6 +72,17 @@ SparseAutoencoder::SparseAutoencoder(const arma::mat& data,
       << "trained model is " << out << "." << std::endl;
 }
 
+inline void SparseAutoencoder::GetNewFeatures(arma::mat& data,
+																							arma::mat& features)
```

Comment:
```suggestion
                                              arma::mat& features)
```
Not sure what happened to this line. :)